sqlair.go

```diff
@@ -124,14 +129,18 @@ func (db *DB) Query(ctx context.Context, s *Statement, inputArgs ...any) *Query
 	return &Query{pq: pq, run: run, ctx: ctx, err: nil}
 }
 
-// Run is an alias for Get that takes no arguments.
+// Run is an alias for Get that takes no arguments. Run is used to run a query
```

Comment:
`Run is an alias for Get that takes no arguments.`  <- Do you think this is an implementation detail? I would just keep the second sentence.

Reply:
It is an implementation detail, but one the user should know about, it makes it obvious whats going on and is very easy to understand. I've swapped the sentences.

Reply:
The only problem would be if we decide to change the inner working of the function in the future because `Run is an alias for [Query.Get] that takes no arguments` involves much more than `Run is used to run a query on a database and disregard any results.`